### PR TITLE
fftw 3.3.6-pl1

### PR DESCRIPTION
**Pull request by fxcoudert**
Created with `brew bump-formula-pr`.

### CODE REVIEW

File: Formula/fftw.rb [hunk 1]
```diff
@@ -1,8 +1,9 @@
 class Fftw < Formula
   desc "C routines to compute the Discrete Fourier Transform"
   homepage "http://www.fftw.org"
-  url "http://www.fftw.org/fftw-3.3.5.tar.gz"
-  sha256 "8ecfe1b04732ec3f5b7d279fdb8efcad536d555f9d1e8fabd027037d45ea8bcf"
+  url "http://fftw.org/fftw-3.3.6-pl1.tar.gz"
+  version "3.3.6-pl1"
```
Comment (woodruffw) [inline Formula/fftw.rb]: This explicit `version` shouldn't be necessary - `brew` can determine the version from the `url`.

Comment (fxcoudert) [inline Formula/fftw.rb]: I tried autodetection first, but without the explicit `version`, `brew` thinks it is version `1`.

The code in `version.rb` clearly says the patch regex is `/p[0-9]*/i`, while here we have 3.3.6-**pl**1.

Comment (ilovezfs) [inline Formula/fftw.rb]: Yeah, it would actually trigger audit failure were the explicit version redundant.

Comment (woodruffw) [inline Formula/fftw.rb]: > Yeah, it would actually trigger audit failure were the explicit version redundant.

Good to know. Ignore this then!

**Comment by dunn** [comment]
Thanks! Merged as af946c7e5d10b86f814e415a675c3d38079a267e.

**Pull request closed without merging**